apps/wallet-dashboard/app/(protected)/layout.tsx

```diff
@@ -26,7 +26,7 @@ function DashboardLayout({ children }: PropsWithChildren): JSX.Element {
             </div>
 
             <div className="container relative flex min-h-screen flex-col">
-                <div className="sticky top-0">
+                <div className="sticky top-0 z-10 bg-gradient-to-b from-gray-100 via-gray-100 to-gray-100/8 dark:from-gray-900 dark:via-gray-900 dark:to-gray-900/8">
```

Comment:
looks good, but I think a backdrop blur, like in explorer, might fit better to keep consistency between apps, wdyt? :pray: